bin/cli.js

```diff
@@ -35,11 +35,29 @@ function checkLocalModule(env) {
   }
 }
 
-function initKnex(env) {
+function mkConfigObj(env, commander) {
+  env.configuration = {
+    ext: 'js',
+    [commander.env || process.env.NODE_ENV || 'development']: {
+      useNullAsDefault: commander.client === 'sqlite3',
+      client: commander.client,
+      connection: commander.connection,
+      migrations: {
+        directory: commander.migrationsDirectory,
+      },
+    },
+  };
+}
+
+function initKnex(env, commander) {
   checkLocalModule(env);
 
-  if (!env.configPath) {
-    exit('No knexfile found in this directory. Specify a path with --knexfile');
+  if (!env.configuration) {
+    if (commander.client) mkConfigObj(env, commander);
```

Comment:
did you run prettier formatting before commit?..

Reply:
No. Is it necessary?